lib/engine/game/g_1841/map.rb

```diff
@@ -0,0 +1,252 @@
+# frozen_string_literal: true
+
+require_relative 'meta'
+require_relative '../base'
+
+module Engine
+  module Game
+    module G1841
+      module Map
+        LAYOUT = :flat
+
+        TILES = {
+          '1' => 1,
+          '2' => 1,
+          '3' => 3,
+          '4' => 3,
+          '5' => 4,
+          '6' => 4,
+          '7' => 8,
+          '8' => 16,
+          '9' => 11,
+          '12' => 1,
+          '13' => 1,
+          '14' => 2,
+          '15' => 2,
+          '16' => 2,
+          '17' => 2,
+          '18' => 2,
+          '19' => 2,
+          '20' => 2,
+          '21' => 2,
+          '22' => 2,
+          '23' => 3,
+          '24' => 3,
+          '25' => 3,
+          '26' => 2,
+          '27' => 2,
+          '28' => 2,
+          '29' => 2,
+          '30' => 2,
+          '31' => 2,
+          '39' => 2,
+          '40' => 2,
+          '41' => 2,
+          '42' => 2,
+          '43' => 2,
+          '44' => 2,
+          '45' => 2,
+          '46' => 2,
+          '47' => 2,
+          '55' => 1,
+          '56' => 1,
+          '57' => 4,
+          '58' => 3,
+          '63' => 3,
+          '69' => 1,
+          '70' => 2,
+          '87' => 2,
+          '88' => 2,
+          '144' => 2,
+          '201' => 2,
+          '202' => 2,
+          '204' => 2,
+          '205' => 1,
+          '206' => 1,
+          '207' => 2,
+          '208' => 2,
+          '216' => 2,
+          # '601' => 1,
+          '602' => 1,
+          '603' => 1,
+          '604' => 1,
+          '605' => 1,
+          '606' => 1,
+          '607' => 1,
+          '609' => 1,
+          '610' => 2,
+          '611' => 3,
+          '612' => 1,
+          '613' => 3,
+          '614' => 4,
+          '615' => 1,
+          '616' => 1,
+          '617' => 1,
+          '618' => 1,
+          '619' => 2,
+          '621' => 2,
+          '622' => 2,
+          '623' => 2,
+          '624' => 1,
+          '625' => 1,
+          '626' => 1,
+          '627' => 2,
+          '628' => 2,
+          '629' => 2,
+          '630' => 1,
+          '631' => 1,
+          '632' => 1,
+          '633' => 1,
+        }.freeze
+
+        LOCATION_NAMES = {
+          A1: 'Lausanne & Lötschberg',
```

Comment:
Nitpick: This can be removed; Lausanne & Loetschberg is below at it's normal location D1

Reply:
Thanks.